src/lib.rs

```diff
@@ -1,13 +1,11 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+// Ideally, this would be handled completely by CI, but there is a bug in `petgraph` that prevents us from doing that.
 #![warn(missing_docs)]
 
-//! TODO
+//! The basic types for Chronicle.
```

Comment:
I think something more fundamental/bold should be written here something a long the line "Core driver library for Chronicle" or something.

Reply:
I just pulled it out of you know where :trollface: Just find something that says a bit more that "basic types" because it also contains the queries and a lot of other logic ... Maybe "Backend library for Chronicle" or something.